example/issue-898/generated.go

```diff
@@ -0,0 +1,3035 @@
+// Code generated by github.com/99designs/gqlgen, DO NOT EDIT.
```

Comment:
you should be able to test this in codegen/testserver, rather than creating a whole new example. examples are for people not tests.

Reply:
ok, removed the example. I have some tests in modelgen, not sure if they are sufficent.